spec/controllers/v0/form214192_controller_spec.rb

```diff
@@ -3,6 +3,56 @@
 require 'rails_helper'
 
 RSpec.describe V0::Form214192Controller, type: :controller do
+  let(:form_data) do
```

Comment:
Maybe we could share json across tests by loading it from a fixture like this
```
let(:form_data) do
    JSON.parse(Rails.root.join('spec', 'fixtures', 'pdf_fill', '21-2680', 'simple.json').read)
  end
```

Or make a factory similarly

Reply:
Done.